makefile.shade

```diff
@@ -16,9 +16,16 @@ k-standard-goals
 #benchmark-tests .compile
   @{
     var sql = string.IsNullOrEmpty(E("BENCHMARK_SQL")) ? string.Empty : "-q " + E("BENCHMARK_SQL");
-    var scenarios = new string[] { "plaintext", "json", "mvcplaintext", "mvcjson" };
+    var scenarios = new string[]
+    {
+      "-n plaintext",
+      "-n plaintext --connectionFilter Benchmarks.PassthroughConnectionFilter",
```

Comment:
The existing convention used in the benchmarks app was camelCase for the config key: https://github.com/aspnet/benchmarks/blob/dev/src/Benchmarks/Program.cs#L133.  Do you think this should be changed, or just the command-line parameters?